src/mavis_config/config.json

```diff
@@ -489,6 +489,11 @@
             "description": "skip the validation stage of the MAVIS pipeline",
             "type": "boolean"
         },
+        "summary.cluster_radius": {
+            "default": 10,
+            "description": "Maximum distance allowed between breakpoint pairs of called SVs and those of known SVs",
+            "type": "integer"
```

Comment:
You should specify a minimum of 0 here so we don't get nonsensical negative inputs

Reply:
I think you may have misunderstood. The default can stay as 10, but you can add a new field "minimum": 0 

see spec docs here https://json-schema.org/understanding-json-schema/reference/numeric.html#range